examples/FCCee/higgs/mH-recoil/histmaker.py

```diff
@@ -0,0 +1,181 @@
+
+# list of processes (mandatory)
+processList = {
+    #'p8_ee_ZZ_ecm240':{'fraction':1},
+    #'p8_ee_WW_ecm240':{'fraction':1}, 
+    #'wzp6_ee_mumuH_ecm240':{'fraction':1},
+    'p8_ee_WW_ecm240':{'fraction':1, 'input': 'localSamples/p8_ee_WW_ecm240/', 'crossSection': 0.25792}, #   16.4385
```

Comment:
Where in histomaker the "input" key is used?

Reply:
Nowhere anymore, this file is obsolete and I'll delete it